packages/types/src/documents/app/table/schema.ts

```diff
@@ -219,3 +219,15 @@ export function isAttachmentField(
 ): field is AttachmentFieldMetadata {
   return field.type === FieldType.ATTACHMENTS
 }
+
+export function isAutoColumnField(
+  field: FieldSchema
+): field is AutoColumnFieldMetadata {
```

Comment:
We should not actually need these wards. If you are doing `field.type === FieldType.AUTO`, the types `AutoColumnFieldMetadata` should be inferred on usage

Reply:
🙈 I feel silly.